scripts/generate-app-config/generatePagesConfig.js

```diff
@@ -136,6 +136,91 @@ const buildPageConfig = (filePaths, generatePagesOptions, namespace) => (
   }, {})
 );
 
+const getScreenshotPatterns = generatePagesOptions => (
+  generatePagesOptions.searchPatterns.reduce((acc, { root }) => {
+    const rootPath = root.replace(/[\\]/g, '/');
+    acc.push({
+      pattern: `${rootPath}/packages/*/tests/wdio/__snapshots__/reference/**/*.png`,
+      // build out a regex for the entrypoint mask.
+      entryPoint: `${rootPath}/packages/[^/]*/tests/wdio/__snapshots__/reference/`,
+    });
+    acc.push({
+      pattern: `${rootPath}/tests/wdio/__snapshots__/reference/**/*.png`,
+      // build out a regex for the entrypoint mask.
+      entryPoint: `${rootPath}/tests/wdio/__snapshots__/reference/`,
+    });
+
+    return acc;
+  }, [])
+);
+
+const createEvidenceParent = packageNamespace => (
+  {
+    name: startCase(packageNamespace),
+    path: `/${kebabCase(packageNamespace)}`,
+    group: '',
+    pages: {},
+  }
+);
+
+const createEvidencePage = (packageNamespace, name) => (
+  {
+    name: startCase(name),
+    path: packageNamespace ? `/${kebabCase(packageNamespace)}/${kebabCase(name)}` : `/${kebabCase(name)}`,
+    group: '',
+    content: {},
+    type: 'screenshot',
+  }
+);
+
+const buildTestEvidenceConfig = (generatePagesOptions, namespace) => {
+  const patterns = getScreenshotPatterns(generatePagesOptions);
+
+  // Execute the globs and regex masks, to trim the directories.
+  const filePaths = patterns.reduce((acc, { pattern, entryPoint }) => (
+    acc.concat(glob.sync(pattern, { nodir: true }).map(filePath => ({ filePath, entryPoint: new RegExp(entryPoint).exec(filePath)[0] })))
+  ), []);
+
+  const combinedPaths = filePaths.reduce((acc, { filePath }) => {
+    // Break up the file path
+    const parsedPath = path.parse(filePath);
+    // Grab the type (doc, test, etc) and the name without the extension.
+    const { name } = parseExtension(parsedPath.name);
+    const directory = parsedPath.dir;
+    const contentPath = relativePath(filePath);
+    const packageNamespace = getNamespace(directory, namespace);
+    const subpath = parsedPath.dir.split('__snapshots__/')[1].split('/');
```

Comment:
I am hesitant on the `/` -- I can't remember. @mjhenkes Are grep'd patterns always returned with `/` separators or do we need to be concerned with windows?

Reply:
Window support is very hard to maintain when we don't have windows computers to test on.